infra/lib/api-stack.ts

```diff
@@ -375,6 +375,47 @@ export class APIStack extends Stack {
       dynamoDBTokenTable,
     });
 
+    // add webhook path for apple health clients
+    const appleHealthResource = webhookResource.addResource("apple");
+    const integrationApple = new apig.Integration({
+      type: apig.IntegrationType.HTTP,
+      options: {
+        connectionType: apig.ConnectionType.VPC_LINK,
+        vpcLink: link,
+      },
+      integrationHttpMethod: "POST",
+      uri: `http://${fargateService.loadBalancer.loadBalancerDnsName}${appleHealthResource.path}`,
+    });
+    appleHealthResource.addMethod("POST", integrationApple, {
+      apiKeyRequired: true,
+    });
+
+    // add another usage plan for Publishable (Client) API keys
+    // everything is throttled to 0 - except explicitely permitted routes
+    const appleHealthThrottleKey = `${appleHealthResource.path}/POST`;
```

Comment:
Can we move these to a function? This main function is super large and is getting hard to understand/maintain.

Reply:
For sure, will update

Reply:
Will address this after merge to unblock staging deployments